packages/playwright-file-snapshots/tests/matcher-config.spec.ts

```diff
@@ -40,3 +40,11 @@ test("disables soft assertions", () => {
     expect("changed value").toMatchJsonFile(),
   ).rejects.toThrowError();
 });
+
+test("applies indentSize to JSON file snapshots", () => {
```

Comment:
You could add a test / extend this one to show nothing changes for text files - but it probably does not matter...

Reply:
Good point, it's always better to explicit test for it in case the behavior is changed in the future. I added the respective tests for both Playwright and Vitest.